src/main/java/jenkins/plugins/logstash/LogstashConfiguration.java

```diff
@@ -211,6 +212,13 @@ public void migrateData()
             logstashIndexer = syslog;
             enabled = true;
             break;
+          case LOGZIO:
```

Comment:
this never existed in old version, so no need to convert old configuration

Reply:
same as above